workers/storagereplication.py

```diff
@@ -220,6 +220,11 @@ def create_gunicorn_worker():
                 has_local_storage = True
                 break
 
+    if app.config.get("DISABLE_PUSHES", False):
+        logger.debug("Pushes to the registry are disabled; skipping startup")
+        while True:
+            time.sleep(100000)
+
```

Comment:
This isn't technically pushing an image, just replicating the storage from one location to another one. 
Is there a specific reason we would want to prevent these types of operations, while allowing things like quota calculations.

Reply:
Removed.